2017-2018/GameEngine/graphics/variables.py

```diff
@@ -0,0 +1,20 @@
+DISPLAY_FREQUENCY = 1/60 # 60 Hz
+SPRITE_FREQUENCY = 6
+GRID_SIZE = (28, 31)
+SQUARE_SIZE = 30
+
+# ghost enums:
+RED = 0
```

Comment:
By no means should you feel a need to use this, but I think you should be aware of it if you were not previously, it's kinda cool: https://docs.python.org/3/library/enum.html

Reply:
I was not aware :O might refactor